IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcLinearZone/Documentation.md

```diff
@@ -0,0 +1,3 @@
+A potentially overlapping linear element.
```

Comment:
Should we say something about the (non-existing) geometry of such a zone?

Reply:
Yes, probably. I will draft something. Any help is appreciated.

Reply:
Drafted something. Please check!